userspace/libscap/scap-int.h

```diff
@@ -56,6 +56,7 @@ struct scap
 	scap_userlist* m_userlist;
 	struct ppm_proclist_info* m_driver_procinfo;
 	uint32_t m_fd_lookup_limit;
+	bool m_minimal_scan;
```

Comment:
I think I'm missing this field initialization to false (except for the `!engine_params || !engine_params->full_proc_scan` case)

Reply:
Or perhaps this is `calloc`'d or already zero initialized in the common part

Reply:
https://github.com/falcosecurity/libs/blob/master/userspace/libscap/scap.c#L557 there

Reply:
Answering myself, we use at least C99 in the scap c part, and the `stdbool.h` header is included by scap. Works for me, I think we can merge it.